org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java

```diff
@@ -258,4 +258,12 @@ protected void assertUploadBinary(String restURL, String fileName) {
 			.then()
 			.statusCode(204);
 	}
+
+	public static String testStringConverter(String toConvert) {
```

Comment:
Name test more precisely. This is NOT a test (not annotated by `@Test`, but something different.)